### DISCUSSION TITLE
Disable automatic preview deployments but still allow triggering via API?

### DISCUSSION THREAD

**Discussion by cj** [vercel/community#233] [Help] [answered]
Hi,

Happy Friday!

How do you disable automatic preview deployments, but still allow them to be triggered via the Vercel API https://vercel.com/docs/rest-api#endpoints/deployments/create-a-new-deployment?

**Comment by samcx** [comment, marked as answer]
Unfortunately, it is not possible to completely disable automatic preview deployments, but you can cancel builds via the [Ignore Build Step](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel) (cancel all builds not equaling your Production Branch).
